stable/prometheus-rabbitmq-exporter/templates/monitor.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.prometheus.monitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "prometheus-rabbitmq-exporter.fullname" . }}
+  labels:
+    app: {{ template "prometheus-rabbitmq-exporter.name" . }}
+    chart: {{ template "prometheus-rabbitmq-exporter.chart" . }}
+    heritage: {{ .Release.Service }}
+  {{- if .Values.prometheus.monitor.additionalLabels }}
+{{ toYaml .Values.prometheus.monitor.additionalLabels | indent 4 }}
+  {{- end }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "prometheus-rabbitmq-exporter.name" . }}
+      release: {{ .Release.Name }}
```

Comment:
Should probably have a namespaceSelector to narrow down to release namespace

Reply:
added namespace to servicemonitor